packages/common/src/ui/components/inputs/CurrencyInput/CurrencyInput.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { FC, useEffect } from 'react';
+import { styled } from '../../../../styles';
+import RCInput, {
+  CurrencyInputProps as RCInputProps,
+} from 'react-currency-input-field';
+
+interface CurrencyInputProps extends RCInputProps {
+  onChangeNumber: (value: number) => void;
+  maxWidth?: number;
+}
+
+// TODO: It would be nice if we were to just use the BasicTextInput or
+// another MUI text input rather than trying to recreate the style so that
+// it could stay in sync with style updated.
+const StyledCurrencyInput = styled(RCInput)(({ theme }) => ({
```

Comment:
given the idea of instead using `currency.js` or similar, maybe not an issue. One thought was to have these styles in a common location and share them across various components

Reply:
I wasn't sure about it.. generally I think most things can jsut use `BasicTextInput` and extend it - unfortuantely this package didn't have an `InputComponent` prop to be able to use it which would have been good. Thought to leave it here (mainly because I wouldn't know where to put them otherwise!) for now... 🤷